mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -304,6 +304,14 @@ func TestAccDataflowFlexTemplateJob_withKmsKey(t *testing.T) {
 	bucket := "tf-test-dataflow-bucket-" + randStr
 	topic := "tf-test-topic" + randStr
 
+	if acctest.BootstrapPSARole(t, "service-", "compute-system", "roles/cloudkms.cryptoKeyEncrypterDecrypter") {
```

Comment:
not an issue w/ this PR, but shouldn't we use the external term ("service agent" ) in the function name rather than a typo of the internal term?

Reply:
That seems reasonable to me.